src/Viewer/Viewer.js

```diff
@@ -215,13 +227,28 @@ export function Viewer ({fileInfo, prettifyLog, logEventNumber, timestamp}) {
                 setStatusMessage("");
                 break;
             case CLP_WORKER_PROTOCOL.LOAD_LOGS:
+                if (resetFolding.current !== undefined) {
```

Comment:
```suggestion
                if (undefined !== addFolding.current) {
```
Again personal preference, but this is the style we've been adopting. (If I don't say this here, Kirk would say this later)